sql_generators/mobile_kpi_support_metrics/__init__.py

```diff
@@ -198,11 +199,11 @@ class MobileProducts(Enum):
     focus_android = Product(
         friendly_name="Focus Android",
         is_mobile_kpi=True,
-        attribution_groups=[AttributionFields.install_source],
+        # attribution_groups=[AttributionFields.install_source],
```

Comment:
This can be added once attribution table is added for these products.

Reply:
It'd recommend removing the comments before merging and add the code in a later PR, when it's decided / implemented.

Reply:
Or make it explicit with a NULL, if possible.

Reply:
Yes, I will remove this. `AttributionFields.empty` is the default hence no need to specify anything here.